kairon/shared/data/collection_processor.py

```
@@ -1,9 +1,225 @@
+
+        :param bot: bot id
+        :return: yield dict
+        """
+        for value in CollectionData.objects(bot=bot):
+            final_data = {}
+            item = value.to_mongo().to_dict()
+            collection_name = item.pop('collection_name', None)
+            is_secure = item.pop('is_secure')
+            data = item.pop('data')
+            data = DataProcessor.prepare_decrypted_data(data, is_secure)
+            final_data["_id"] = item["_id"].__str__()
```

Pylint (3.3.7)

[convention] Unnecessarily calls dunder method __str__. Use str built-in function.

(C2801)